component/admin/views/product_price/view.html.php

```diff
@@ -28,7 +28,7 @@ public function display($tpl = null)
         $db->setQuery($sql);
         $product = $db->loadObject();
 
-        $sql = "SELECT g.*,p.product_price,p.price_id,p.price_quantity_end,p.price_quantity_start FROM #__redshop_shopper_group g LEFT JOIN #__redshop_product_price p ON g.shopper_group_id = p.shopper_group_id   AND product_id = '$productId'";
+        $sql = "SELECT g.*,p.product_price,p.price_id,p.price_quantity_end,p.price_quantity_start FROM #__redshop_shopper_group g LEFT JOIN #__redshop_product_price p ON g.id = p.shopper_group_id   AND product_id = '$productId'";
```

Comment:
please change this query and above query to use Joomla query

Reply:
done